fastlane/Fastfile

```diff
@@ -127,9 +265,6 @@ before_all do |lane|
   # Skip these checks/steps for test lane (not needed for testing)
   next if lane == :test_without_building
 
-  # Ensure we use the latest version of the toolkit
-  check_for_toolkit_updates unless is_ci || ENV['FASTLANE_SKIP_TOOLKIT_UPDATE_CHECK']
-
```

Comment:
I'm guessing this was removed during development? We should restore it, though.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/commit/75334b7c2ba858e1da781ee0d0f95c0ab7805d06